apps/web/src/server/app/app.router.js

```diff
@@ -32,7 +33,7 @@ if (!config.authDisabled) {
 }
 
 router.route('/').get(viewHomepage);
-router.route('/auth/signout').get(handleSignout);
+router.route('/auth/signout').get(asyncRoute(handleSignout));
 router.route('/documents/:caseId/upload').post(postDocumentsUpload);
```

Comment:
I didn't add the asyncRoute everywhere because in some case it was generating issues, but I'm definetely going to do that in a specific ticket for that.

Reply:
ok - if that's not creating issues we can review it later